assertk-common/src/test/kotlin/test/assertk/assertions/CollectionTest.kt

```diff
@@ -8,11 +8,11 @@ import kotlin.test.assertFails
 
 class CollectionTest {
     //region isEmpty
-    @Test fun isEmpty_empty_passes() {
+    @Test fun isEmptyPasses() {
```

Comment:
The underscores here were on purpose, it makes it easier to read when the message is long, which can be quite common in tests. Originally It'd be something like \`isEmpty passes\`(), but that doesn't work when targeting js ☹️.

Reply:
Ok, i'll revert it.